canopen/objectdictionary/__init__.py

```diff
@@ -13,6 +13,20 @@
 logger = logging.getLogger(__name__)
 
 
+def export_od(od, dest=None, docType="eds"):
+    """ Export :class: ObjectDictionary to a file.
+    :param docType: type of document to export
+    :rtype: str or None
+    """
```

Comment:
The docstring doesn't correlate with the actual interface here.

Reply:
What is says is, i think, true but i admit that is says far too little.  I will expand it a little.